packages/ckeditor5-table/ckeditor5-metadata.json

```diff
@@ -153,6 +153,22 @@
 				"table.contentToolbar",
 				"table.tableToolbar"
 			]
+		},
+		{
+			"name": "Table caption",
+			"className": "TableCaption",
+			"description": "Adds support for table captions, which inform the reader about the content of the table. Using captions is also beneficial from the accessibility point of view.",
+			"docs": "features/table.html#table-caption",
+			"path": "src/tablecaption.js",
+			"requires": [
+				"Table"
+			],
+			"htmlOutput": [
+				{
+					"elements": "figcaption",
+					"attributes": "data-placeholder"
```

Comment:
I don't see the `data-placeholder` attribute in the generated HTML after calling `editor.getData()` from editor containing table with table caption. Is this an editing view attribute?

Reply:
Oh, it's used in the editing downcast :facepalm: . Thanks. I'll fix it temporarily in the OB and I'll fix it before the next release..